test/unit/format_capture_helper_test.rb

```diff
@@ -82,6 +86,10 @@ def assert_captured_content(content, format, expected)
       should "not wrap empty content with govspeak html tags" do
         assert_captured_content("", :govspeak, "")
       end
+
+      should "remove extra trailing newlines from html" do
+        assert_captured_content("<div>content</div>\n\n\n\n", :html, "<div>content</div>")
```

Comment:
Where do these come from and how come we are concerned? can they be sorted at the source at all?

Reply:
Agreed - the templates themselves should just be formatted properly. Not an existing problem as HTML format is new, so I've removed this behaviour.